package.json

```diff
@@ -97,17 +98,5 @@
     "node": ">=0.10"
   },
   "license": "MIT",
-  "main": "public/main.js",
-  "window": {
-    "title": "Ungit",
-    "icon": "icon.png",
-    "toolbar": false,
-    "frame": false,
-    "show": false,
-    "width": 1000,
-    "height": 600,
-    "position": "center",
-    "min_width": 400,
-    "min_height": 200
-  }
+  "main": "electron/main.js"
```

Comment:
I'm not sure about this line, I think it may point npm commands to electron folder and may result difficulty in debugging as execution will not be in the `public`

I will need to read up more and test this

Reply:
Good catch :+1: this point is tricky. Electron Application Architecture guide (https://electronjs.org/docs/tutorial/application-architecture) says that electron runs the script present at `main` inside the package.json

I see 2 possibilities : 
1. Keep the configuration as it is with an electron specific file hidden in `public/main.js` (previous configuration)
2. Make an electron folder to name things and mode `public/main.js` here to clearly states it is for electron (what I chose)

The rest of ungit always run the `./bin/ungit` binary and is not affected by this change

Reply:
I haven't done the packaging node projects and I'm not too certain on this matter but I think previous option makes more sense as how we package our app should not affect how the app should be built.